db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 comedy = Category.create(title: 'Comedy')
 action = Category.create(title: 'Action')
 
-Video.create(title: 'Inception', 
+inception = Video.create(title: 'Inception', 
```

Comment:
I have found the discussion about `create` and `create!` (http://stackoverflow.com/questions/6316821/rails-create-and-create-methods-ror-3-tutorial), it seems that the `create!` will throw a exception instead of just a false return when create fail, but what's the benefit to add a ! here?